packages/preview/modern-cv/0.2.0/typst.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "modern-cv"
+version = "0.2.0"
+compiler = "0.11.0"
+entrypoint = "lib.typ"
+authors = ["Paul Tsouchlos <https://github.com/DeveloperPaul123>"]
+license = "MIT"
+description = "A modern resume template based on the Awesome-CV Latex template."
+keywords = ["CV", "Curriculum Vitae", "Resume", "Minimalistic", "Professional", "Modern"]
+categories = ["cv"]
+disciplines = []
+exclude = ["resume-documentation*", "scripts*"]
```

Comment:
The only assets which seems to be used by the package itself is `square-phone-solid.svg`. The image assets for the README should thus be excluded here to keep the package size down.